src/applications/mhv/secure-messaging/reducers/threads.js

```diff
@@ -65,3 +65,53 @@ export const threadsReducer = (state = initialState, action) => {
       return state;
   }
 };
+
+export const messageThreadsReducer = (state = initialState, action) => {
```

Comment:
Would you please elaborate on the purpose of this reducer export?

Reply:
This was a mock test for my learning purposes and shouldn't have gone in. Please disregard.😬